README.md

```diff
@@ -1,143 +1,134 @@
-<div align="center">
-
-<img height="250" width="250" src="./assets/Badge_Trdelnik.png"/>
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-cli?label=CLI)](https://crates.io/crates/trdelnik-cli)
```

Comment:
I'm thinking if it's not better to place it under Trdelnik logo

Reply:
I like how the Anchor repo looks like, and it is under Logo, but it just inspirational
https://github.com/project-serum/anchor

Reply:
:facepalm: I think I was quite confused yesterday .. I thought logo is the name of some folder and you want from me to place the badges in this folder :smile: Moved .. sorry